cmr/queries.py

```diff
@@ -832,7 +892,7 @@ def granule_ur(self, granule_ur: str) -> Self:
         return self
 
     def readable_granule_name(
-        self, readable_granule_name: Union[str, Sequence[str]]
+            self, readable_granule_name: Union[str, Sequence[str]]
```

Comment:
I think extra indentation slipped in via another PR:

```suggestion
        self, readable_granule_name: Union[str, Sequence[str]]
```